CHARTER.md

```diff
@@ -0,0 +1,104 @@
+# MapLibre Charter
+
+## Definitions
+
+*MapLibre Organization:* An umbrella organization to develop, maintain and promote MapLibre software projects
+
+*MapLibre Projects:* multiple repositories, each being some part of the map visualization stack from the raw data to the map visualization and interaction.
+
+*MapLibre:* As a single word, “MapLibre” is a brand name used by the front-end map rendering MapLibre projects (i.e. code libraries).
+
+*MapLibre Community:* People interested in the MapLibre Projects. Anyone can be part of the MapLibre Community and there are no special requirements except to follow community code of conduct. For example, the Community members … :
+
+ * evangelize the MapLibre Project
```

Comment:
This follows the definitions, but already uses a non-defined term.

Did you mean "MapLibre Organization" / "MapLibre Projects" / "MapLibre Community" / ...?
Right now, this could be understood as one of the "MapLibre Projects" (follow up questions: which one?).

Reply:
> MapLibre Organization

I believe this one. @wipfli can correct me if I'm wrong. I think it's just a holdover from a previous draft. :)

Reply:
Yes

Reply:
Might also be an English mistake. @nyurik google says these would be examples to use the word "evangelize": 
> The missionaries set out to evangelize the world.
> They were evangelizing about the importance of saving energy.

Reply:
Rephrased it...